paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -48,27 +59,31 @@ class AccessibilityRenderExtension : RenderExtension {
     val canvasImage = BufferedImage(image.width * 2, image.height, image.type)
 
     val graphics = canvasImage.createGraphics()
+    graphics.font = robotoFont.deriveFont(DEFAULT_TEXT_SIZE)
     graphics.drawImage(image, 0, 0, null)
     graphics.color = DEFAULT_DESCRIPTION_COLOR
     graphics.fillRect(image.width, 0, image.width, image.height)
 
     val colorRectSize = DEFAULT_RECT_SIZE
-    graphics.font = graphics.font.deriveFont(DEFAULT_TEXT_SIZE)
-
     val drawStartX = image.width + image.margin()
     accessibilityTree.forEachIndexed { index, view ->
       val drawStartY = (index * colorRectSize * 1.5f).toInt() + image.margin()
 
-      graphics.color = getColor(view)
-      graphics.fillRoundRect(drawStartX, drawStartY, colorRectSize, colorRectSize,
-          colorRectSize / 4, colorRectSize / 4)
+      val viewColor = getColor(view)
+      graphics.color = viewColor
+      graphics.fillRoundRect(
+        drawStartX, drawStartY, colorRectSize, colorRectSize,
+        colorRectSize / 4, colorRectSize / 4
+      )
       graphics.color = DEFAULT_TEXT_COLOR
       val accessibilityText = view.iterableTextForAccessibility.toString()
 
-      view.createAccessibilityNodeInfo()
-      graphics.drawString(accessibilityText,
-          drawStartX + colorRectSize + image.margin(),
-          drawStartY + colorRectSize / 2 + graphics.fontMetrics.height / 2)
+      val lineMetrics = graphics.fontMetrics.getLineMetrics(accessibilityText, graphics)
+      graphics.drawString(
+        accessibilityText,
+        (drawStartX + colorRectSize + image.margin()).toFloat(),
+        drawStartY + colorRectSize / 3 + lineMetrics.height / 2f
```

Comment:
some subtle changes here, is this calling a different function overload now?  also, should 3 now be 3f?

Reply:
ya, there were some subtle changes to the font metrics measurement. I can revert some of this back though.